src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoReadOnly.cs

```diff
@@ -12,17 +12,26 @@ public class DateTimeFormatInfoReadOnly
         public static IEnumerable<object[]> ReadOnly_TestData()
         {
             yield return new object[] { DateTimeFormatInfo.InvariantInfo, true };
+            yield return new object[] { DateTimeFormatInfo.ReadOnly(new DateTimeFormatInfo()), true };
```

Comment:
Nit: it'd be good to have an entry here where we're calling ReadOnly on a DateTimeFormatInfo that's not already read-only